### DISCUSSION TITLE
Clickhouse cleaning from spam requests

### DISCUSSION THREAD

**Discussion by ezitisitis** [plausible/analytics#4602] [Self-Hosted Support]
Maybe someone had doing that before. 
I want to make a little cleaning of analytics from trash requests. As I understand I can just delete rows from `imported_pages`, but what about other tables?

**Comment by ruslandoga** [comment]
👋 

imported_pages is probably not the table you are looking for :) Spam would probably end up in events_v2 and sessions_v2.
